config/default.yml

```diff
@@ -990,6 +990,11 @@ Style/LambdaCall:
     - braces
 
 Style/MethodCallWithArgsParentheses:
+  Exclude:
```

Comment:
Yeah, it would be nice to get this cop working the way we want and apply it to our projects. These exclusions would exclude a bunch of the code I write. 😢 

My concern is the potential surprise and confusion from users when the same code triggers this cop from some parts of their app, but not others. OTOH, perhaps we should release and see if that really happens instead of speculating.